.github/workflows/rust.yml

```diff
@@ -166,6 +166,8 @@ jobs:
         run: |
           rustup update ${{ matrix.toolchain }} && rustup default ${{ matrix.toolchain }}
           rustup component add rustfmt
+      - name: Check all
+        run: cargo check --tests --benches --all-features --workspace
```

Comment:
Aren't we about to do `cargo build` anyways?  Why run check first?  Earlier failure detection?

Reply:
That's fair. I've merged this into the build so we aren't doing things twice.